tasks.py

```diff
@@ -0,0 +1,261 @@
+import git
+import json
+import os
+import sys
+
+from shutil import copyfile
+from invoke import task
+
+
+APP_NAME = "fecfile-web-app"
+ORG_NAME = "fec-fecfileonline-prototyping"
+APP_BUILD_NAME = "fec-eFilling"
+
+
+def _detect_space(repo, branch=None):
+    """Detect space from active git branch.
+    :param str branch: Optional branch name override
+    :returns: Space name if space is detected and confirmed, else `None`
+    """
+    space = _resolve_rule(repo, branch)
+    if space is None:
+        print("The current configuration does not require a deployment to cloud.gov.   ")
+        return None
+    print("Detected space {space}".format(**locals()))
+    return space
+
+
+def _detect_prod(repo, branch):
+    """Deploy to production if master is checked out and tagged."""
+    if branch != "master":
```

Comment:
Let's make this `main`, this is my oversight.

Reply:
Changed